RocketChat/Rocket.Chat.ReactNative
# Chore: Migrate SelectServerView to Typescript

## Proposed changes

Migrate SelectServerView to Typescript

## Issue(s)	


## How to test or reproduce

- To test this, you must be logged into two servers
- Then, share a file or text, and the Share App will open and you should see a section written as `Select server`
- Click on the server, and you will be moved to the view migrated.
- 
## Screenshots

## Types of changes




## Checklist



## Further comments
